spec/features/donor_edits_location_spec.rb

```diff
@@ -8,7 +8,6 @@
     new_pickup_location = {
       address: "123 Fake Street Denver CO",
       zipcode: zone.zipcode,
-      notes: "on my porch",
     }
```

Comment:
@desi it looks like Corwin's going to handle it soon: https://github.com/thoughtbot/freshfoodconnect/pull/42#issuecomment-209132508.

Good to merge?